libmcount/mcount.c

```diff
@@ -585,6 +591,45 @@ void mcount_entry_filter_record(struct mcount_thread_data *mtdp,
 
 }
 
+static void script_hook_exit(struct mcount_thread_data *mtdp,
+			     struct mcount_ret_stack *rstack)
```

Comment:
Could you please move it close to the `script_hook_entry`?

Reply:
Yep !